### PR TITLE
Liwo 109 add search

### PR DESCRIPTION
**Pull request by StefanKool**

### CODE REVIEW

File: src/components/SearchInput.vue [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<script>
+import { mapbox } from 'mapbox.js'
+
+// import 'mapbox.js/theme/style.css'
```
Comment (markomalis) [inline src/components/SearchInput.vue]: remove comment

File: src/components/BaseLayerControl.vue [hunk 2]
```diff
@@ -1,55 +1,33 @@
...
-  props: [ 'tileLayers' ],
-  methods: {
-    selectBaseLayer (event) {
-      this.$emit('baselayer', event.target.value)
-    }
+  render () {
+    return ''
+  },
+  props: [ 'tileLayers', 'mapRef' ],
+  mounted () {
```
Comment (markomalis) [inline src/components/BaseLayerControl.vue]: think we should move this to a computed prop in the parent component

File: src/components/LiwoMap.vue [hunk 3]
```diff
@@ -18,23 +17,27 @@
       :continuous-world="continuousWorld"
       :attribution="attribution"
     />
+    <search-input :mapRef="mapRef" />
+    <l-control-zoom></l-control-zoom>
     <base-layer-control
       :tile-layers="baseLayer.tileLayers"
-      @baselayer="updateBaseLayer"
+      :mapRef="mapRef"
```
Comment (SiggyF) [inline src/components/LiwoMap.vue]: `map-ref`

File: src/components/LiwoMap.vue [hunk 4]
```diff
@@ -18,23 +17,27 @@
       :continuous-world="continuousWorld"
       :attribution="attribution"
     />
+    <search-input :mapRef="mapRef" />
```
Comment (SiggyF) [inline src/components/LiwoMap.vue]: map-ref

**Pull request closed without merging**